api/README.md

```diff
@@ -1,52 +1,73 @@
 # Kitty Items API
+The Kitty Items API is a RESTful API built with [express](https://expressjs.com/) that sends transactions to Flow using the [Flow JS SDK](https://github.com/onflow/fcl-js/tree/master/packages/sdk). It contains endpoints for both the [kitty-items](src/services/kitty-items.ts) & [storefront](src/services/storefront.ts) services to read & write data from both the Flow blockchain as well as a SQL DB.
 
-The Kitty Items API is a RESTful API built with [express](https://expressjs.com/) that sends transactions to Flow using the [Flow JS SDK](https://github.com/onflow/fcl-js/tree/master/packages/sdk).
+## Services
+Kitty Items uses main 4 services to function. These services are used to maintain data integrity between the blockchain & our database as well as allow avenues for users to interact with our data through exposed api endpoints. 
 
-## Setup
+### [Flow Service](src/services/flow.ts)
+This service contains functions that interact with the Flow blockchain using the [onflow/fcl](https://docs.onflow.org/fcl/) package. While it has no exposed endpoints, the methods here used to read, write & authorize data on the chain are used extensively in its sister services & workers.
```

Comment:
```suggestion
This service contains functions that interact with the Flow blockchain using the [FCL JS](https://docs.onflow.org/fcl/) library. While it has no exposed endpoints, its methods are used to read, write and authorize data on the chain are used extensively by its sister services and workers.
```

IMO best to only use ampersands (&) in titles

Reply:
`While it has no exposed endpoints, its methods ~are~ used to read, write and authorize data on the chain are used extensively by its sister services and workers.`